tests/duration.rs

```diff
@@ -577,3 +577,21 @@ fn test_min_max() {
     assert_eq!(d1, d1.max(d0));
     assert_eq!(d1, d0.max(d1));
 }
+
+#[test]
+fn regression_test_gh_244() {
+    use hifitime::Epoch;
+
+    let _local0 = if let Ok(x) = Epoch::maybe_from_gregorian_utc(1792, 1, 1, 0, 0, 0, 0) {
+        x
+    } else {
+        std::process::exit(0);
+    };
+    let _local5 = Epoch::duration_in_year(&_local0);
```

Comment:
For all of these variables, could you remove the leading `_`? A leading underscore is to tell the compiler to ignore the fact that a given variable isn't used. All of the variables in the tests should be used, so we don't want those to show up.